forge-gui/res/adventure/Shandalar/custom_cards/cursed_treasure.txt

```diff
@@ -3,6 +3,7 @@ ManaCost:no cost
 Types:Artifact
 S:Mode$ Continuous | Description$ Provided by Cursed Treasure (Equipped Item - Right)
 StackDescription$ Create a Treasure token. You lose 2 life. | SpellDescription$ Create a Treasure token. You lose 2 life.
-A:AB$ Token | Cost$ PayShards<1> Sac<1/CARDNAME> | TokenScript$ c_a_treasure_sac | SubAbility$ DBLoseLife2 | SpellDescription$ Create a Treasure token.
+A:AB$ Token | Cost$ PayShards<1> T | ActivationZone$ Command |  TokenScript$ c_a_treasure_sac | SubAbility$ DBLoseLife2 | SubAbility$ Eject | SpellDescription$ Create a Treasure token.
```

Comment:
tap cost in command zone?

Reply:
Intentional: Tap cost in command zone prevents the single use card from being triggered again in response to something else being placed on the stack. Card is placed in a tapped state (and will not ever untap without an ability specifically doing so) even though it doesn't appear that way on the UI, and thus that cost cannot be paid again.

{M}, Exile Example Item: Target creature gets a +1/+1 token.
Example item actual script: {M} {T}: Target creature gets a +1/+1 token. Exile Example Item.

Player activates, example item targeting a 3/3.
Opponent interrupts by targeting the 3/3 with a lightning bolt for 3 damage.
Player cannot spend an extra mana shard to put another copy of this ability on top of the stack and have the creature live, and it's the tap cost that prevents it.